pkgs/misc/drivers/openzwave/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub
+, doxygen, libxml2, git, which, pkgconfig, graphviz-nox
+, udev }:
+
+let
+  version = "2018-04-04";
+
+in stdenv.mkDerivation rec {
+  name = "openzwave-${version}";
+
+  src = fetchFromGitHub {
+    owner = "OpenZWave";
+    repo = "open-zwave";
+    rev = "ab5fe966fee882bb9e8d78a91db892a60a1863d9";
+    sha256 = "0yby8ygzjn5zp5vhysxaadbzysqanwd2zakz379299qs454pr2h9";
+  };
+
+  nativeBuildInputs = [ doxygen libxml2 git which pkgconfig graphviz-nox ];
+  buildInputs = [ udev ];
+
+  preBuild = ''
+    makeFlagsArray+=("DESTDIR=$out")
+    makeFlagsArray+=("PREFIX=$out")
+    makeFlagsArray+=("pkgconfigdir=lib/pkgconfig")
+    makeFlagsArray+=("sysconfigdir=etc/openzwave")
```

Comment:
You can set this in `makeFlags`, no need to use `preBuild`. See https://nixos.org/nixpkgs/manual/#build-phase.

Reply:
That is much nicer, thanks :+1:

Reply:
It doesn't seem to work like the docs suggests:

```
  makeFlags = [
    "DESTDIR=$(out)"
    "PREFIX=$(out)"
    "pkgconfigdir=lib/pkgconfig"
    "sysconfigdir=etc/openzwave"
  ];
```

Doesn't build and it seems like the issue is with `$out` because it outputs this:
```
build flags: SHELL=/nix/store/q1g0rl8zfmz7r371fp5p42p4acmv297d-bash-4.4-p19/bin/bash DESTDIR=\$\(out\) PREFIX=\$\(out\) pkgconfigdir=lib/pkgconfig sysconfigdir=etc/openzwave
```

Reply:
Doesn't help at all :D

Reply:
I tried myself and it gives me a different error when leaving out parantheses: `mv: cannot stat '/nix/store/nhy5v0l6yccgi2wfp01d457dp9g6ql9w-openzwave-2018-04-04//nix/store/nhy5v0l6yccgi2wfp01d457dp9g6ql9w-openzwave-2018-04-04/bin': No such file or directory`.

Reply:
Yeah, that's the fixupPhase